src/components/simple-popper-modal/simple-popper-modal.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useRef } from 'react'
+import cn, { Argument } from 'classnames'
+
+import { useOutsideClick } from '@core/hooks/use-outside-click'
+
+export interface ISimplePopperModalProps {
+  close: () => void
```

Comment:
May be `onClose`

Reply:
It leads to deeper refactoring and causes too many side-changes for a single PR. 'close' handler is created in a custom hook and it is used in various places among the project.